README.md

```
@@ -1,5 +1,5 @@
 <div align="center">
-<img src="./docs/assets/images/logo/cruiseKube_Colour.png" width="200">
+<img src="./docs/assets/images/logo/variants/cruisekube-stacked-color.svg" width="200">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)